include/qdmi/constants.h

```diff
@@ -807,6 +814,169 @@ enum QDMI_DEVICE_PULSE_SUPPORT_LEVEL_T {
 /// Pulse support level type.
 typedef enum QDMI_DEVICE_PULSE_SUPPORT_LEVEL_T QDMI_Device_Pulse_Support_Level;
 
+/// Enum of the telemetry sensor properties that can be queried via @ref
+/// QDMI_device_session_query_telemetrysensor_property as part of the @ref
+/// device_interface "device interface" and via @ref
+/// QDMI_device_query_telemetrysensor_property as part of the @ref
+/// client_interface "client interface".
```

Comment:
```suggestion
/**
 * Enum of the telemetry sensor properties that can be queried via @ref
 * QDMI_device_session_query_telemetrysensor_property as part of the @ref
 * device_interface "device interface" and via @ref
 * QDMI_device_query_telemetrysensor_property as part of the @ref
 * client_interface "client interface".
 */
```
I'd prefer this style for multi-line comments (also for consistency).